CHANGELOG.md

```diff
@@ -45,6 +45,10 @@ backend {
 }
 ```
 
+### Womtool endpoint
+
+When submitting a workflow together with an inputs JSON to the `/describe` endpoint, the response now differentiates between an invalid workflow and a valid workflow with invalid inputs. Look for the `inputErrors` key in the response or see the Swagger for more information.
```

Comment:
> Look for the `inputErrors` key in the response

ToL: Is `inputErrors` always present now? So presence of the *key* does not automatically mean there exists an error? The returned list has to be non-empty, right?

Reply:
Yes. For better or for worse we are doing "all keys always present". I know this is a source of perennial debate and if there is a consensus to change we can.